RELEASE_NOTES.md

```diff
@@ -26,6 +26,8 @@ An example `.eslintrc` can be found [here](org.eclipse.wildwebdeveloper.tests/te
 
 For more information on configuring ESLint, visit https://eslint.org/docs/user-guide/configuring.
 
+▶️ Demo of ESLint in Wild Web Developer https://youtu.be/o-wI_niEz3E
```

Comment:
This line might be better suited at the top of the text, right below the header? It's your call @mickaelistria